hack/fix_cmx_swagger

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+
+
+# This is the location of the CMX swagger
+old_filename = "swagger/vra-cmx.json"
+new_filename = "swagger/vra-cmx-fixed.json"
+
+
+def replace_value(d, k, v, new):
+    if k in d and d[k] == v:
+        print(d[k])
```

Comment:
Remove debug messages

Reply:
this is a copy & paster from the other scripts, I'll remove this and clean up the other scripts.